Source/inv.cpp

```diff
@@ -829,7 +829,7 @@ void CheckInvCut(int pnum, Point cursorPosition, bool automaticMove, bool dropIt
 			if (automaticMove) {
 				if (CanBePlacedOnBelt(holdItem)) {
 					automaticallyMoved = AutoPlaceItemInBelt(player, holdItem, true);
-				} else {
+				} else if (CanEquip (holdItem)) {

```

Comment:
Don't forget to run clang-format
```suggestion
				} else if (CanEquip(holdItem)) {
```